.github/workflows/fortress-tracker.yml

```
@@ -0,0 +1,35 @@
+          python scripts/update-legacy-tracking.py
+
+      - name: Commit updates (if any)
+        run: |
+          git config --local user.email "action@github.com"
+          git config --local user.name "Fortress Tracker"
+          git add LEGACY_TRACKING.md
+          git diff --staged --quiet || git commit -m "auto: update legacy tracking data"
+
+      - name: Check fortress violations
+        run: |
+          python scripts/fortress-guard.py
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)